src/frame/endpoints/PolicyApiEndpoints.ts

```diff
@@ -0,0 +1,56 @@
+import {ErrorCodes} from "../../Constants";
+import * as ApiRequest from "../ApiRequest";
+import ApiState from "../ApiState";
+import Future from "futurejs";
+import {SDKError} from "../../shim";
+import {UserOrGroup, Policy} from "ironweb";
+
+export type UserOrGroupWithKey = UserOrGroup & {masterPublicKey: PublicKey<string>};
+
+interface PolicyApplyResponse {
+    usersAndGroups: UserOrGroupWithKey[];
+    invalidUsersAndGroups: UserOrGroup[];
+}
+
+/**
+ * Generate signature message from current user state
+ */
+function getSignatureHeader() {
+    const {segmentId, id} = ApiState.user();
+    return ApiRequest.getRequestSignature(segmentId, id, ApiState.signingKeys());
+}
+
+/**
+ * Transform one of the edeks for the logged in user.
+ * @param {Uint8Array} edeks The encrypted deks to send.
+ */
+export function applyPolicy(req: Policy) {
```

Comment:
an alternative name here could be `evalPolicy`

Reply:
We call it policy application elsewhere, so just trying to be consistent.